s3transfer/download.py

```diff
@@ -24,7 +24,7 @@
     ReadTimeoutError
 
 from s3transfer.compat import SOCKET_ERROR
-from s3transfer.compat import seekable
+from s3transfer.compat import seekable, writes_are_seekable
```

Comment:
Just a fly-by comment here, but unless I'm missing something, you don't actually call this `writes_are_seekable()` function anywhere, so how does this fix the problem?

Reply:
Excellent that someone is awake, added the missing call from my staging 👍